### DISCUSSION TITLE
Waiter Check list to customer

### DISCUSSION THREAD

**Discussion by otakuventure** [Blair2004/NexoPOS#1447] [Gastro 4.x Ideas & Issues]
**Describe the bug**
Looking for a Checklist for waiter for table dine in customer

**To Reproduce**
None Bugs Related

**Expected behavior**
1. waiter key in Order
2. POS to printout checklist to customer 
3. Upon order is server, waiter to tick the box

**Screenshots**
![image](https://user-images.githubusercontent.com/69811457/232949664-4b00093e-07c9-4f5f-b919-cd722226e352.png)

**Environment:**
Core Version | 4.8.17
Laravel Version | 9.52.5

**Comment by Blair2004** [comment]
Hi, can you elaborate on this ? Is that something that helps the waiter knows what was served?

**Comment by otakuventure** [comment]
Hi, 

Yes, and normally this check list would be printed and placed at customers table for table service customer. 

Upon food arrived at the table, waiter will cross out this those delivered item and client can take note those item has been served.

**Comment by Blair2004** [comment]
Hi,
We have to discuss how important this feature is... Some will instead see a waste of paper. However, I do appreciate the contribution. I'll put this thread aside so that we can reconsider this, based on users thoughts.
